src/index.html

```diff
@@ -2,16 +2,190 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      name="form"
+    >
+      <fieldset class="box__style">
+        <legend>Personal information</legend>
+
+        <div class="field__wrapper">
+          <label for="surname">Surname: </label>
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            minlength="3"
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="name">Name: </label>
+
+          <input
+            type="text"
+            name="name"
+            id="name"
+            minlength="3"
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="age">How old are You? </label>
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="date">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+
+        <label for="check">I accept the term of the agreement </label>
+
+        <input
+          type="checkbox"
+          value="check"
+          name="check"
+          id="check"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>Registration</legend>
+
+        <div class="field__wrapper">
+          <label for="email">E-mail: </label>
+
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            multiple
+          >
+        </div>
+
+        <label for="user-password">Password: </label>
+
+        <input
+          name="user-password"
+          id="user-password"
+          type="password"
+          required autocomplete="off"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="field__wrapper">
+          <label for="cat">Do you love cats? </label>
+
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+            value="Y"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+            value="N"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field__wrapper">
+          <label for="color">What is your favorite color? </label>
+
+          <input
+            type="color"
+            name="color"
+          >
```

Comment:
Label for this field is not clickable(when clicking label it should focuse on the input).
Add **id** which coresponds with **for** attribute in label
```suggestion
          <input
            type="color"
            name="color"
            id="color"
          >
```